app/controllers/competitions_controller.rb

```diff
@@ -10,10 +10,7 @@ def shark_tank
   end
 
   def go_fish
-    @breadcrumbs = [
-        { text: 'Home', path: root_path },
-        { text: 'GoFish!' }
-    ]
+    redirect_to root_path
```

Comment:
❓ Not sure if we need this... or if I can remove the `/competitions/go-fish` route entirely and a user who happens to navigate to `/competitions/go-fish` wouldn't get a weird error